app/_data/kong_versions.yml

```diff
@@ -389,7 +389,11 @@
   endOfLifeDate: "2025-02-01"
   branch: release-2.6
 - edition: mesh
-  version: preview
+  version: 2.7.0
   release: 2.7.x
+  branch: release-2.7
```

Comment:
Is 2.7.x the new latest? If so, we should set latest: true to it, and remove that bit from 2.6.x

Reply:
2.7.x is not yet released, I will update it in a separate PR, after we release this version